starport/pkg/clispinner/clispinner.go

```diff
@@ -60,9 +67,8 @@ func (s *Spinner) Start() *Spinner {
 // Stop stops spinning.
 func (s *Spinner) Stop() *Spinner {
 	s.sp.Stop()
```

Comment:
@Pantani you mean calling a lock here causes to a deadlock? To be precise, where do you release it? If it is after `UpdateCharSet` it is expected because that method also acquires a lock.

Reply:
Ohh, you are right, I didn't pay attention to these locks, sorry!! 
I updated the PR